sample-libs/Samples.DatabaseHelper/DatabaseTestHarness.cs

```diff
@@ -0,0 +1,272 @@
+using System;
+using System.Data;
+using System.Data.Common;
+using System.Linq;
+using System.Threading.Tasks;
+using Datadog.Trace;
+
+namespace Samples.DatabaseHelper
+{
+    public class DatabaseTestHarness<TConnection, TCommand, TDataReader>
```

Comment:
I *love* this.
Should we name this something like `RelationalDatabaseTestHarness`?
As not all of these concepts will share between other database types like key stores or document based.

Reply:
Renamed in 0ffda7e6c81b28b31e45e7919dce0052b8b750be.